packages/ilp-spsp-payout/src/index.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { PayoutConnection } from './lib/PayoutConnection'
+import { pluginFromEnvironment as makeILPPlugin } from 'ilp-plugin'
+
+const CLEANUP_TIMEOUT = 30 * 1000
+
+export class Payout {
+  private createPlugin: Function
+  private payouts: {
+    [pointer: string]: {
+      connection: PayoutConnection,
+      lastSent: number,
+      timer: NodeJS.Timer
+    }
+  }
+
+  constructor (makePlugin?: any) {
```

Comment:
I think this should go into an opts object rather than passing in positionally. So like

```js
new Payout({ makePlugin })
```

also type should not be `any`